docs/_static/network/v1.yaml

```diff
@@ -211,6 +211,21 @@ paths:
                   $ref: '#/components/schemas/EventSubscriber'
         default:
           $ref: '../common/error_response.yaml'
+  /internal/network/v1/addressbook:
```

Comment:
I don't think so, since we plan to release this in v5.2 and we could just fix the other change in v5.2 as well. And even then, the things to change are an addition, so they're backwards compatible.

Reply:
As long as the other issue is fixed as well its fine I guess.